tests/test_augmentations.py

```diff
@@ -83,10 +83,12 @@ def test_image_only_augmentations_with_float_values(augmentation_cls, params, fl
     [CenterCrop, {'height': 10, 'width': 10}],
     [RandomCrop, {'height': 10, 'width': 10}],
     [RandomSizedCrop, {'min_max_height': (4, 8), 'height': 10, 'width': 10}],
+    [RandomCenteredBBoxCrop, {'max_part_shift': 0.15}]
 ])
 def test_dual_augmentations(augmentation_cls, params, image, mask):
     aug = augmentation_cls(p=1, **params)
-    data = aug(image=image, mask=mask)
+    annotations = {'image': image, 'cropped_bbox': [-59, 77, 177, 231]}
```

Comment:
you should not change tests, just add another if existing does not fit your needs.